### ISSUE TITLE
Make changes to multiple complaints at once

### ISSUE THREAD

**Issue by lalitha-jonnalagadda** [usdoj-crt/crt-portal-management#452]

**Comment by lalitha-jonnalagadda** [comment]
- Respond to multiple people at once from the view all table
- Make changes to status at once
- Select an outcome for multiple complaints

**Comment by blaughman** [comment]
Bulk processing took care of many user needs.  Bulk email responses still to be developed.  New card will be created once that feature is ready to be developed.  Closing this card.
